bot_example/src/unindexed_events.rs

```diff
@@ -0,0 +1,84 @@
+use std::{collections::BTreeMap, error::Error};
+
+use aderyn_driver::context::loader::ContextLoader;
+use aderyn_driver::detection_modules::capture;
```

Comment:
![Screenshot 2024-01-10 at 13 41 34](https://github.com/Cyfrin/aderyn/assets/6523673/79c35933-18bd-4b49-a4e3-251d771cb994)

Everything builds fine, but I'm getting this in my vscode. The same with `capture!`. Do you get the same?